SECURITY.md

```diff
@@ -12,9 +12,9 @@ The specification is a living document and is supported for two major versions
 unless otherwise noted.
 
 > Security reports for the version 1 algorithm will not be accepted. It has
-> a well-known issue and exists solely to provide support to already existing
-> apps until they can be upgraded. A future version of the specification will
-> shift from _recommending_ against the use of version 1 to actively
+> a well-known lifetime issue and exists solely to provide support to already
+> existing apps until they can be upgraded. An unscheduled future specification
+> version of the will shift from _recommending_ against the use of version 1 to
```

Comment:
Should this be "An unscheduled future version of the specification will shift…"?

(nit) In my mind there’s a difference between "unscheduled" and "not yet scheduled"